src/foundations/shape/Shape.tsx

```diff
@@ -26,13 +23,7 @@ const Shape = ({
         true
     );
 
-    const rootProperties = {
-        radius,
-        elevation,
-        elevationOnHover,
-    };
-
-    return <ShapeRoot as={element} {...rootProperties} {...rest} />;
+    return <ShapeRoot as={element} radius={radius} {...rest} />;
```

Comment:
we need to revert this now to allow `elevation` and `elevationOnHover` again.

Reply:
I am sorry I missed that